src/components/table/DataTableProperties.js

```diff
@@ -1,14 +1,15 @@
+import { LinkExternal } from '@styled-icons/boxicons-regular'
 import styled from 'styled-components/macro'
 
 import { RESOURCES } from '../entry/resources'
+import { theme } from '../ui/GlobalStyle'
 
 const TableLinkPreview = styled.img`
   width: 20px;
   height: 20px;
   margin-right: 5px;
 `
 const NameLink = styled.a`
-  font-size: 1rem !important;
   font-weight: normal;
   color: ${({ theme }) => theme.secondaryText} !important;
```

Comment:
I don't know if this was because you needed to override something, but I think typically !important is bad practice.